module/journal.js

```diff
@@ -116,16 +124,24 @@ export default class WorldAnvilBrowser extends Application {
 
       // Get the category to which the article belongs
       const category = categories.get(article.category?.id) || uncategorized;
-      category.articles.push(article);
+      category.unsortedArticles.push(article);
     }
 
     // Sort articles within each category
     for ( let category of categories.values() ) {
-      category.articles.sort( (a,b) => a.title.localeCompare(b.title) );
+      category.articles = category.articleIds
+        .map( id => category.unsortedArticles.find( a => a.id === id )  )
+        .filter( a => !!a );
```

Comment:
Instead of iterating twice with `.map` and `.filter` it would be better to do a single `.reduce()`